source/developers/minio-drivers.rst

```diff
@@ -12,113 +12,75 @@ Software Development Kits (SDK)
 
 MinIO publishes the following Software Development Kits (SDK):
 
-1. :ref:`.NET <dotnet-sdk>`
-2. :ref:`Golang <go-sdk>`
-3. :ref:`Haskell <haskell-sdk>`
-4. :ref:`Java <java-sdk>`
-5. :ref:`JavaScript <javascript-sdk>`
-6. :ref:`Python <python-sdk>`
-7. :ref:`C++ <cpp-sdk>`
+- :ref:`Go <go-sdk>`
+- :ref:`Python <python-sdk>`
+- :ref:`Java <java-sdk>`
+- :ref:`.NET <dotnet-sdk>`
+- :ref:`JavaScript <javascript-sdk>`
+- :ref:`C++ <cpp-sdk>`
+- :ref:`Haskell <haskell-sdk>`
 
-.. _cpp-sdk:
-
-C++ (``minio-cpp``)
--------------------
-
-
-Reference
-  `MinIO C++ SDK Reference <https://minio-cpp.min.io/>`__
+.. _go-sdk:
 
-Install
-  - ``vcpkg``
+Go
+--
 
-    .. code-block:: shell
-       :class: copyable
+GitHub: `minio/minio-go <https://github.com/minio/minio-go>`__
 
-       vcpkg install minio-cpp
+Latest Version: |go-sdk-version|
 
-  - Source
-             
-    .. code-block:: shell
-       :class: copyable
+Quickstart Guide: :doc:`/developers/go/minio-go`
 
-       git clone https://github.com/minio/minio-cpp
-       cd minio-cpp
-       wget --quiet -O vcpkg-master.zip https://github.com/microsoft/vcpkg/archive/refs/heads/master.zip
-       unzip -qq vcpkg-master.zip
-       ./vcpkg-master/bootstrap-vcpkg.sh
-       ./vcpkg-master/vcpkg integrate install
-       cmake -B ./build -DCMAKE_BUILD_TYPE=Debug -DCMAKE_TOOLCHAIN_FILE=./vcpkg-master/scripts/buildsystems/vcpkg.cmake
-       cmake --build ./build --config Debug
-
-.. _dotnet-sdk:
-
-.NET (``minio-dotnet``)
------------------------
+Download from GitHub
+  .. code-block:: shell
+     :class: copyable
 
-Latest Version
-  |dotnet-sdk-version|
+     go install github.com/minio/minio-go/v7@latest
 
-Reference
-  :doc:`MinIO .NET SDK Reference </developers/dotnet/minio-dotnet>`
 
-Download from NuGet
-  Run the following command in the NuGet Package Manager Console.
+.. _python-sdk:
 
-  .. code-block:: shell
-     :class: copyable
+Python
+------
 
-     PM> Install-Package Minio
+GitHub: `minio/minio-py <https://github.com/minio/minio-py>`__
 
-.. _go-sdk:
+Latest Version: |python-sdk-version|
 
-Go (``minio-go``)
------------------
+Quickstart Guide: :doc:`/developers/python/minio-py`
```

Comment:
We do also have the reference. We could point to both here consistently.

Reply:
oo yes